llm/vicuna/serve.yaml

```diff
@@ -1,6 +1,7 @@
 resources:
   accelerators: A100:1
   disk_size: 1024
+  disk_tier: high
```

Comment:
add: Note: requires installing SkyPilot from source (<= 0.2.5 does not have this flag)